EIPS/eip-2537.md

```diff
@@ -0,0 +1,293 @@
+---
+eip: 2537
+title: BLS12-381 curve operations
+author: Alex Vlasov (@shamatar)
+discussions-to: https://github.com/ethereum/EIPs/pull/2537
```

Comment:
The current EIP-1 rules state this can't be a link to a PR.

Reply:
Updated